packages/storage-plugin/src/storage.plugin.ts

```diff
@@ -13,7 +13,8 @@ export class NgxsStoragePlugin implements NgxsPlugin {
 
   handle(state, event, next) {
     const options = this._options || <any>{};
-    const isInitAction = getActionTypeFromInstance(event) === '@@INIT';
+    const actionType = getActionTypeFromInstance(event);
+    const isInitAction = actionType === '@@INIT' || actionType === '@@UPDATE_STATE';
```

Comment:
Can the init stuff go in the construction rather then checking for the init event?